src/index.scss

```diff
@@ -1,7 +1,18 @@
 @import './partials/calculator';
 @import './partials/buttons';
 @import './partials/result';
+@import './partials/navbar';
+@import './partials/home';
+@import './partials/quote';
 
 body {
   margin: 0;
+  overflow: auto;
+  background-color: #c53117;
+
```

Comment:
- [ ] optional

I strongly suggest that you deploy your website for visitior to have proper access to the website 😄